<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWin</class>
 <widget class="QMainWindow" name="AuthWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuthWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Записать на мероприятие</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Просмотреть мероприятия</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
